daily-challenges/2024-11/13/ththao272.spec.ts

```
@@ -0,0 +1,25 @@
+    await page.click('//button[@onclick="startGame()"]');
+    await expect(page.locator('#playerInfo')).toHaveText(`Người chơi: ${playerName}`);
+
+    const texts = await page.locator('//div[@class="outer"]').allInnerTexts();
+    let setOuter: Set<string> = new Set(texts);
+
+    page.on('dialog', dialog => {
+        expect(dialog.message()).toEqual('Bạn đã thắng cuộc!');
+        dialog.accept();
+    });
+
+    for (let value of setOuter) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'value' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)